### DISCUSSION TITLE
Any tool to convert generated HCL into ERD?

### DISCUSSION THREAD

**Discussion by chenrui333** [ariga/atlas#1288] [General]
I was trying to search and see if there is any way to convert the HCL code into ERD, but did not find anything useful, so raise this up to see if anybody has ideas of doing this. Let me know if that also makes sense. Thanks!

**Comment by a8m** [comment]
Hey @chenrui333! We just discussed this today internally, and decided to add support for it really soon 🙂

**Reply by chenrui333** [reply]
lucky day for me 😄

**Reply by chenrui333** [reply]
Also another thing, should we create a feature request issue to track down this?

**Reply by rotemtam** [reply]
hey @chenrui333 , in case you missed it:

https://gh.atlasgo.cloud/explore is it :D
